src/contributing.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: layouts/base.njk
+---
+
+Welcome! First off; _**THANK YOU**_ for your interest in contributing to Zinc!
+Your time is valuable, so let's get started!
+
+## Ground Rules
+
+Zinc moves forward thanks to two kinds of workers. _Contributors_, who suggest
+and build changes to our software; and _Maintainers_, who decide if, when, and
+how to apply the suggestions. Anyone and everyone can be a Contributor! All we
+ask is that Contributors follow a few [Ground Rules][contributor-ground-rules]
+to help Zinc Maintainers and Contributors work well together.
+
+The keywords _must_, _must not_, and _should_ in this document are to be
+interpreted as described in [RFC 2119][rfc-2119].
+
+### Contributor Ground Rules
+
+- Contributors _must_ comply with Zinc's [code of conduct][code-of-conduct].
+- Contributors _must_ [assign ownership][assign-ownership] of their
+  contributions to Zinc.
+- Contributors _should_ [ask for help][get-help] when they get stuck.
+- Contributors _should_ expect others to build their Contributions.
+
+### Maintainer Ground Rules
+
+- Maintainers _must_ comply with Zinc's [code of conduct][code-of-conduct].
+- Maintainers _must_ ensure that all feedback provided to Contributors is
+  specific, actionable, and kind.
+- Maintainers _must not_ erase authorship information from any Contributors.
+- Maintainers _should_ respond to Contributors' issues, comments, and patches
+  within two weeks.
+
+## How can I contribute...
+
+### Features?
+
+Gaps between what a Contributor _wishes_ a product did and what it does is a
+_feature_. Small, incremental features with clear use cases are the easiest and
```

Comment:
I think that's actually the opposite of a feature

Reply:
Good catch, it's a "missing" feature. Will add.